package.json

```diff
@@ -119,6 +119,8 @@
     "collectCoverage": true
```

Comment:
Oh these snuck in and shouldn't be here! You can revert them back by doing something like:

```
# First update your master branch
# then,
git checkout master package-lock.json
git checkout master package.json
```

This should take the version of these files currently on master, effectively "undoing" the changes in your branch! Save these on a separate commit :+1: